ARMTemplateInstructions.md

```diff
@@ -0,0 +1,95 @@
+---
+title: ARM template to create Experimentation setup 
+description: How to use CLI and ARM template to create Experimentation setup
+author: muksvso    
+ms.author: mubatra
+ms.service: azure-app-configuration
+ms.topic: how-to 
+ms.date: 22/03/2024
+ms.custom: template-how-to
+---
+
+# Use ARM template for Experimentation resources
+
+This article will help you set up the Experimentation pipeline depicted in Figure (). You will be using Azure ClI and ARM template for majority of the setup.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/dotnet). Owner or Contributor and UserAdminAccess roles in the Azure subscription. This subscription should be whitelisted by Microsoft Product team to run Experimentation.
+
+- You will need [Azure CLI](/cli/azure/install-azure-cli) installed. Sign in to Azure using the `az login` command in Azure CLI. This command will prompt your web browser to launch and load an Azure sign-in page. If the browser fails to open, use device code flow with `az login --use-device-code`. For more sign in options, go to [sign in with the Azure CLI](/cli/azure/authenticate-azure-cli).
+
+---
+
+## Enterprise app for Data access policy
+
+First step is to create and configure the Entra ID's Enterprise App setup. This Entra ID app is used for accessing the Data plane of the Split SaaS from App Configuration and Split Experimentation Workspace.
+
+- To create the Enterprise app registration for data access from Split
+  
+	```azurecli-interactive
+	        az ad app create --display-name <appRegistrationName>
+	```
+- Output will show objectid in the oauth2Permissions. Copy the value for the object id and run the following command with the copied object id.
+  
+	```azurecli-interactive
+		az ad sp create --id <objectid>
+	```
+- Check for the created application registration in **Entra Id -> App Registrations** on Azure portal. Go to **Manifest** blade, update the 'isEnabled' on line 35 to false, save the manifest. Copy the following values from the manifest,  `id` at line2, `appId` at line 7 and `name` at line 26, keep these values for use in next steps.

```

Comment:
Remove "Go to **Manifest** blade, update the 'isEnabled' on line 35 to false, save the manifest"